code/modules/mob/living/carbon/xenomorph/castes/boiler/abilities_boiler.dm

```diff
@@ -62,6 +62,39 @@
 		button.overlays += image('icons/mob/actions.dmi', button, "toggle_bomb0")
 	return ..()
 
+
+/datum/action/xeno_action/toggle_bomb_advanced
```

Comment:
hmmm, the thing about radial menus just is that their icons don't work well with the screen gets scaled, and I imagine boilers would like to be able to swap ammo type during the zoom. I guess I could add this as an alternate option simillar to how Defiler can cycle them via either simple cycling or via radials and just leave it to players?

For the list itself, I guess I could just hijack the spit_types list since boilers don't normally spit and this would avoid having to add another list viable for boiler ammo types.

Reply:
Boilers now have their available globs in the spit_types, radial selection added (uses normal toggle if you have 2 or less globs), including a keybind, normal toggle improved a bunch.